### PR TITLE
HRINT-1468 Sample data not created for the User DB + Upgrade to Rave…

### PR DESCRIPTION
**Pull request by Danielle9897**
…nDB Client 4.2.6

### CODE REVIEW

File: DemoCommon/Utils/Database/Operations/CreateSampleDataOperation.cs [hunk 1]
```diff
@@ -13,7 +14,7 @@ public RavenCommand GetCommand(DocumentConventions conventions, JsonOperationCon
             return new CreateSampleDataCommand();
         }
 
-        private class CreateSampleDataCommand : RavenCommand
+        private class CreateSampleDataCommand : RavenCommand, IRaftCommand
```
Comment (ayende) [inline DemoCommon/Utils/Database/Operations/CreateSampleDataOperation.cs]: why is this needed?

Comment (arekpalinski) [reply DemoCommon/Utils/Database/Operations/CreateSampleDataOperation.cs]: Each cluster related command needs to provide unique ID. We have introduced it in https://issues.hibernatingrhinos.com/issue/RavenDB-13556 and all our commands in client API implements it. Also `CreateSampleDataCommand ` in Tests:

https://github.com/ravendb/ravendb/blob/8fa6b32cc74a0033604972e8d789ce1bb2a96c16/test/Tests.Infrastructure/CreateSampleDataOperation.cs#L17

Comment (ayende) [reply DemoCommon/Utils/Database/Operations/CreateSampleDataOperation.cs]: Oh, because we are setting the revisions in the handler, okay.
